pkgs/tools/archivers/undmg/default.nix

```
@@ -1,30 +1,47 @@
-{ lib, stdenv, fetchFromGitHub, zlib, bzip2, lzfse, pkg-config }:
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  zlib,
+  bzip2,
+  lzfse,
+  xz,
+  pkg-config,
+}:
 
-stdenv.mkDerivation rec {
-  version = "1.1.0";
+stdenv.mkDerivation {
   pname = "undmg";
+  version = "0-unstable-2024-08-02";
```

According to the [recommendations for package versions](https://github.com/NixOS/nixpkgs/tree/master/pkgs)

> If a package is a commit from a repository without a version assigned, then the version attribute should be the latest upstream version preceding that commit, followed by -unstable- and the date of the (fetched) commit. The date must be in "YYYY-MM-DD" format

this should probably be:
```suggestion
  version = "1.1.0-unstable-2024-08-02";
```